requirements-dev.txt

```diff
@@ -4,4 +4,5 @@ mypy
 types-requests
 types-setuptools
 typing_extensions
-types-PyYAML
\ No newline at end of file
+types-PyYAML
+pre-commit
```

Comment:
i'm guessing pre-commit was added because you discovered it was a dev dependency?